lib/drivers/tandemTslimDriver.js

```diff
@@ -0,0 +1,839 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2014, Tidepool Project
+ * 
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ * 
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ * 
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+var _ = require('lodash');
+var struct = require('./../struct.js')();
+var sundial = require('sundial');
+
+module.exports = function (config) {
+    var cfg = config;
+
+    var SYNC_BYTE = 0x55;
+
+    var RESPONSES = {
+        81: {
```

Comment:
I know this is incomplete, but why is this a raw number rather than VERSION_TE ?

If you're just trying to index by number, a helper function that searches for the responses by value would probably be clearer.